README.md

```diff
@@ -10,7 +10,52 @@ This is a Command Line Interface (CLI) designed to simplify the process of setti
 By using this CLI, you can streamline the node setup process and get started with your Madara project quickly and efficiently.
 
 ## Usage
-- git submodule update --init --recursive
-- cargo run create
+```bash
+git submodule update --init --recursive
+```
+### Running in cli mode
+You usually will know what mode you want to run. For example, to run Devnet:
+
+```bash
+madara create devnet
+```
+To learn more, checkout the help command:
+```
+madara --help
+```
+This results in:
+```
+Madara CLI to easily spin up nodes
+
+Usage: madara [OPTIONS] [COMMAND]
+
+Commands:
+  create        Create a Madara node
+  orchestrator  Run Orchestrator ---> AppChain
```

Comment:
Orchestrator command will be merged into create. The user can select AppChain to spin up all the necessary services + orchestrator